src/applications/personalization/profile/components/ProfileInfoTable.jsx

```diff
@@ -60,6 +60,7 @@ const ProfileInfoTable = ({
     'line-height--4',
     'margin--0',
     'margin-bottom--1',
+    'width--auto',
```

Comment:
Hmm this is needed?

Reply:
Unfortunately yes otherwise the dfn is limited to 208px. If you know another way around it I'm all ears

Reply:
I do have to make some adjustments to tests so if you do have another way that would be better to handle this let me know

Reply:
I don't. Just wanted to make sure this wasn't left over from some initial exploratory work

Reply:
ok excellent. Wanted to be sure there wasn't some other way I should have handled it.